services/fluxmonitor_test.go

```diff
@@ -21,12 +21,15 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-// does not print big.int pointers
 func prettyPrint(i interface{}) string {
-	s, _ := json.MarshalIndent(i, "", "\t")
-	return string(s)
+	// does not print big.int pointers
+	// s, _ := json.MarshalIndent(i, "", "\t")
+	// return string(s)
```

Comment:
Can we get rid of this?

```suggestion
```

Optionally, can we get rid of this entire function and just use `fmt.Sprintf("%+v", i)` directly?